onnx-ecosystem/inference_demos/fasterRCNN_object_detection_onnxruntime_inference.ipynb

```diff
@@ -255,7 +255,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "session = onnxruntime.InferenceSession('faster_rcnn_R_50_FPN_1x.onnx')"
+    "# Start from ORT 1.10, ORT requires explicitly setting the providers parameter if you want to use execution providers\n",
+    "# other than the default CPU provider (as opposed to the previous behavior of providers getting set/registered by default\n",
+    "# based on the build flags) when instantiating InferenceSession.\n",
+    "# Following code assumes NVIDIA GPU is available, you can specify other execution providers or don't include providers parameter\n",
+    "# to use default CPU provider.\n",
+    "session = onnxruntime.InferenceSession('faster_rcnn_R_50_FPN_1x.onnx', providers=['CUDAExecutionProvider'])"
```

Comment:
this notebook installs onnxruntime package and not onnxruntime-gpu so even if GPU is available on the system we are not really using it since the package is not built with CUDA

Reply:
Modified it.